python/cudf/cudf/_libxx/issorted.pyx

```diff
@@ -0,0 +1,42 @@
+from cudf._libxx.cudf cimport *
+from cudf._libxx.cudf import *
+from cudf._libxx.utils cimport *
+from cudf._libxx.utils import *
+
+from cudf._libxx.includes.issorted cimport is_sorted
+
+def issorted(Table source_table, order=[], null_prec=[]):
```

Comment:
is this `order` argument shadowing the `order` enum?

Reply:
It would be a list of `bool` that is converted to a vector of `cudf::experimental::order`. So, I think yes.

Reply:
why not `is_sorted`?

Reply:
`column_order` is used elsewhere.

Reply:
Sorry, my comment is _so_ ambiguous. What I meant to say is that elsewere we name this argument "`column_order`" to prevent name collision.